test/system/060-mount.bats

```diff
@@ -151,4 +150,31 @@ load helpers
     run_podman rm -f $cid
 }
 
+@test "podman mount external container - basic test" {
+    # Only works with root (FIXME: does it work with rootless + vfs?)
+    skip_if_rootless "mount does not work rootless"
+    skip_if_remote "mounting remote is meaningless"
+
+    f_path=/tmp/tmpfile_$(random_string 8)
+    f_content=$(random_string 30)
```

Comment:
Neither of these are used; please remove

Reply:
Love cut and paste errors.